pkg/web3/sdk.go

```diff
@@ -229,7 +238,8 @@ func NewContractSDK(options Web3Options) (*Web3SDK, error) {
 		TransactOpts: transactOpts,
 		Contracts:    contracts,
 	}
-	log.Debug().Msgf("Public Address: %s", web3SDK.GetAddress())
+	// fmt.Printf("Public Address: %s\n", web3SDK.GetAddress())
```

Comment:
Left over commented code

Reply:
Thank you @narbs91, I removed the commented code.